test/onerror/inside-test.js

```diff
@@ -33,4 +33,20 @@ QUnit.module( "QUnit.onError", function() {
 
 		assert.strictEqual( result, true, "onError should not allow other error handlers to run" );
 	} );
+
+	QUnit.test( "Should resume test runner if test was stopped", function( assert ) {
+		assert.expect( 0 );
+
+		// This would hold up the test runner, but onError should resume it
+		assert.async();
+
+		// Avoid assertion error
+		QUnit.config.current.pushFailure = function() {};
+
+		QUnit.onError( {
```

Comment:
Can we make this actually async and add an assertion? Something simple like:

```js
setTimeout( function() {
  assert.ok( true, "waited for async" );
  QUnit.onError( .. );
}, 50 );
```

Just to ensure that we actually stopped and then resumed.